torchtext/data/iterator.py

```diff
@@ -252,6 +252,29 @@ def create_batches(self):
                                 sort_within_batch=self.sort_within_batch)
 
 
+def generate_iterators(datasets, batch_sizes, **kwargs):
+    """Create Iterator objects for multiple splits of a dataset.
+
+    Arguments:
+        datasets: a tuple of Datasets
+        batch_sizes: a list of batch_size, corresponding to each dataset.
+        ngrams: the number of ngrams.
```

Comment:
This is specific to text, but the function has potential for being generic by itself. Is there a way to split out this "ngrams" concept?

Reply:
It seems a typo. I don't think iterator function needs ngrams.